zerver/webhooks/flock/doc.md

```diff
@@ -0,0 +1,19 @@
+Get Zulip notifications from your Flock channels.
+
+1. {!create-stream.md!}
+
+1. {!create-bot-construct-url-indented.md!}
+
+1. Click on **Apps** in the bottom-right corner.
+
+1. Click on **Admin Panel**, and click on **Webhooks**.
+
+1. Next to **Outgoing Webhook**, click on **Add**.
```

Comment:
Combine these three points into one, the three after the macros! :)

Reply:
@eeshangarg Okay, so I've combined this in the same way you suggested in the other one :)